api/app/src/routes/util.ts

```diff
@@ -54,46 +54,34 @@ export const getFromParamsOrFail = (prop: string, req: Request): string => {
   return value;
 };
 
-export const getCxId = (req: Request): string | undefined => req.cxId;
+export const getCxId = (req: Request): string | undefined => {
+  const cxId = req.cxId;
+  Sentry.setUser({ id: cxId });
+  return cxId;
+};
 export const getCxIdOrFail = (req: Request): string => {
   const cxId = getCxId(req);
   if (!cxId) throw new BadRequestError("Missing cxId");
   return cxId;
 };
 
-/** @deprecated use getFromQuery() */
-export const getCxIdFromQuery = (req: Request): string | undefined =>
-  req.query.cxId as string | undefined;
-/** @deprecated use getFromQueryOrFail() */
+export const getCxIdFromQuery = (req: Request): string | undefined => {
+  const cxId = req.query.cxId as string | undefined;
+  Sentry.setUser({ id: cxId });
+  return cxId;
+};
 export const getCxIdFromQueryOrFail = (req: Request): string => {
   const cxId = getCxIdFromQuery(req);
   if (!cxId) throw new BadRequestError("Missing cxId query param");
   return cxId;
 };
 
-/** @deprecated use getFromQuery() */
-export const getFacilityIdFromQuery = (req: Request): string | undefined =>
-  req.query.facilityId as string | undefined;
-/** @deprecated use getFromQueryOrFail() */
-export const getFacilityIdFromQueryOrFail = (req: Request): string => {
-  const facilityId = getFacilityIdFromQuery(req);
-  if (!facilityId) throw new BadRequestError("Missing facilityId query param");
-  return facilityId;
-};
-
-/** @deprecated use getFromQuery() */
-export const getPatientIdFromQuery = (req: Request): string | undefined =>
-  req.query.patientId as string | undefined;
-/** @deprecated use getFromQueryOrFail() */
-export const getPatientIdFromQueryOrFail = (req: Request): string => {
-  const patientId = getPatientIdFromQuery(req);
-  if (!patientId) throw new BadRequestError("Missing patientId query param");
-  return patientId;
+export const getCxIdFromHeaders = (req: Request): string | undefined => {
+  const cxId = req.header("cxId") as string | undefined;
+  Sentry.setUser({ id: cxId });
```

Comment:
In all places where this is being used -> `capture.setUserId()`?

Reply:
What do you mean? The way this should work is:
- we set the user ID on Sentry for the request (here)
- until the request is completed, the user ID is part of the context to be sent to Sentry should an error/message occur

Surely, this is about to be tested.

Reply:
All I mean is that we're using `Sentry.setUser({ id: cxId });` here, shouldn't we use our wrapper here instead? ie `capture.setUserId(cxId)`

Reply:
oh, gotcha. good call!

Reply:
Done 👍